### ISSUE TITLE
"Alt text" missing at a lot of places in the documentation causing bugs

### ISSUE THREAD

**Issue by cnstt** [napari/napari#4501]
:books: Documentation

There was a bug In the [Dimension sliders section](https://napari.org/tutorials/fundamentals/viewer.html#dimension-sliders) of the viewer tutorial that has been fixed (#4472), but I have now seen that this bug happens at a lot of other places:
- [x] https://napari.org/tutorials/fundamentals/viewer.html#viewer-buttons
- [x] https://napari.org/tutorials/fundamentals/viewer.html#changing-viewer-theme
- [x] https://napari.org/howtos/layers/image.html#adjusting-contrast-limits
- [x] https://napari.org/howtos/layers/labels.html#a-simple-example
- [x] https://napari.org/howtos/layers/labels.html#d-rendering-of-labels
- [x] https://napari.org/howtos/layers/points.html#a-simple-example
- [x] https://napari.org/howtos/layers/points.html#setting-edge-or-face-color-with-a-color-cycle
- [x] https://napari.org/howtos/layers/points.html#setting-edge-or-face-color-with-a-colormap
- [x] https://napari.org/howtos/layers/shapes.html#a-simple-example
- [x] https://napari.org/howtos/layers/shapes.html#adding-new-shapes
- [x] https://napari.org/howtos/layers/surface.html#a-simple-example
- [x] https://napari.org/howtos/layers/vectors.html#a-simple-example
- [x] https://napari.org/guides/magicgui.html#annotating-as-a-layer-subclass

**Comment by brisvag** [comment]
Just getting to this after seeing #4502. I'm not faimilar with the whole alt-text endevour, but surely simply passing an empty string should not result in an error! While it's useful to have alt-text in the tutorials, the fix should probably be to solve this parse error in the first place. What is the error exactly?

**Comment by cnstt** [comment]
You are right, but alt text is important for accessibility I think? At least that's what I understood it is used for :) So to raise an error when there is not alt text is not necessarily a bad thing, as it forces us to write a small alt text! My PR #4502 is close to being finished so it doesn't take a lot of time :)

```
---------------------------------------------------------------------------
ParserError                               Traceback (most recent call last)
Input In [2], in <cell line: 3>()
      1 from napari.utils import nbscreenshot
----> 3 nbscreenshot(viewer, alt_text=" ")

File ~/work/napari/napari/napari/utils/notebook_display.py:73, in NotebookScreenshot.__init__(self, viewer, canvas_only, alt_text)
     71 self.canvas_only = canvas_only
     72 self.image = None
---> 73 self.alt_text = self._clean_alt_text(alt_text)

File ~/work/napari/napari/napari/utils/notebook_display.py:89, in NotebookScreenshot._clean_alt_text(self, alt_text)
     85 alt_text = html.unescape(
     86     str(alt_text)
     87 )  # cleaner won't recognize unescaped script tags
     88 cleaner = Cleaner()
---> 89 doc = document_fromstring(alt_text)
     90 alt_text = cleaner.clean_html(doc).text_content()
     91 # alt_text = html.escape(alt_text)

File /opt/hostedtoolcache/Python/3.9.12/x64/lib/python3.9/site-packages/lxml/html/__init__.py:763, in document_fromstring(html, parser, ensure_head_body, **kw)
    761 value = etree.fromstring(html, parser, **kw)
    762 if value is None:
--> 763     raise etree.ParserError(
    764         "Document is empty")
    765 if ensure_head_body and value.find('head') is None:
    766     value.insert(0, Element('head'))

ParserError: Document is empty
```
